tools/auto_tuner/src/tune_nn.cpp

```diff
@@ -0,0 +1,48 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2018 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename tune_nn.cpp
+ *
+ **************************************************************************/
+
+#include <cstdlib>
+
+#include "gemm_tuner.hpp"
+
+int main(int argc, char *argv[]) {
+  if (argc != 6) {
+    std::cerr << "Usage: " << argv[0] << " M K N bs rep" << std::endl;
+    return -1;
+  }
+
+  const bool transA = true;
```

Comment:
Wouldn't it be best to just set these to false here, instead of inverting them on line 45?